### DISCUSSION TITLE
Where can I find more information about using EAMxx?

### DISCUSSION THREAD

**Discussion by rljacob** [E3SM-Project/E3SM#6657] [EAMxx help]
Where can I find more information about using EAMxx?

**Comment by PeterCaldwell** [comment]
- EAMxx user guide: https://docs.e3sm.org/scream/user/
- Guide to Doubly-periodic mode: https://github.com/E3SM-Project/scmlib/wiki/E3SM-Intensive-Observation-Period-Case-Library
- Data from Simulation Campaigns is available only once we submit the first paper about the topic. We are working on adding these runs to https://docs.e3sm.org/e3sm_data_docs/_build/html/index.html, but in the meantime you can find data at:
1. SCREAM-Fortran DYAMOND2: On NERSC HPSS at /home/t/terai/Production_runs/SCREAMv0.SCREAM-DY2.ne1024pg2.20201127/ 
2. SCREAM-C++ 4 seasons output: On NERSC HPSS at /home/a/adonahue/www/ and via the web at at https://portal.nersc.gov/archive/home/a/adonahue/www/ 

In the works:
- Guide to regionally refined mode
- Developer guide
- Example run scripts
- Landing page providing all this info at e3sm.org
